### ISSUE TITLE
property_exists on object always returns true

### ISSUE THREAD

**Issue by flack** [midgardproject/midgard-php5#32]
with the latest git, this code:

```
$t = new midgard_topic();
var_dump(property_exists($t, 'thispropertydoesnotexist'));
```

returns true.

if you changes this to 

```
var_dump(property_exists('midgard_topic', 'thispropertydoesnotexist'));
```

it returns false like it should

**Comment by bergie** [comment]
Some remarks on how this should work:

```
$topic = new midgard_topic();
property_exists($topic, 'name'); // true
property_exists($topic, 'foo'); // false
$topic->bar = 'baz';
property_exists($topic, 'bar'); // true
```

**Comment by indeyets** [comment]
do not return true for non-existant properties. Closed by 354998af9c0884003d01210e4e47bb31ba8aee6e
